app/addons/documents/assets/less/viewEditor.less

```diff
@@ -10,76 +10,223 @@
 // License for the specific language governing permissions and limitations under
 // the License.
 
-@import "../../../../../assets/less/animations.less";
 @import "../../../../../assets/less/variables.less";
 
-.keyframes(fadeInDownNoReduce, {
-    opacity: 0;
-    height: 0px;
-},
-{
-    opacity: 1;
-    height: 546px;
-});
-
-
-.keyframes(fadeInDownReduce, {
-    opacity: 0;
-    height: 0px;
-},
-{
-    opacity: 1;
-    height: 745px;
-});
-
-.keyframes(fadeOutUpReduce, {
-    opacity: 1;
-    height: 745px;
-},
-{
-    opacity: 0;
-    height: 0px;
-});
-
-.keyframes(fadeOutUpNoReduce, {
-    opacity: 1;
-    height: 546px;
-},
-{
-    opacity: 0;
-    height: 0px;
-});
-
-
-.fadeInDownNoReduce-enter {
- .animation(fadeInDownNoReduce 1s both);
-}
+.editor-wrapper {
 
-.fadeInDownNoReduce-leave {
- .animation(fadeOutUpNoReduce 1s both);
-}
 
-.fadeInDownReduce-enter {
- .animation(fadeInDownReduce 1s both);
-}
+  .define-view {
+    padding-bottom: 70px;
+  }
+
+  .define-view {
+    .help-link {
+      margin-left: 3px;
+    }
+  }
+  label {
+    font-size: 16px;
+  }
+  .bordered-box {
+    border-bottom: 1px solid #ccc;
+  }
+  .padded-box {
+    margin: 25px 30px;
+  }
+  .db-title {
+    color: @brandPrimary;
+    line-height: 30px;
+  }
+  .new-ddoc-input {
+    margin-top: 25px;
+  }
 
-.fadeInDownReduce-leave {
- .animation(fadeOutUpReduce 1s both);
+  .styled-select {
+    width: 250px;
+  }
+  .styled-select label {
+    margin: 0;
+  }
+  .styled-select select {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    background-color: #e6e6e6;
+    border: 1px solid #b3b3b3;
+    height: 45px;
+    width: 250px;
+  }
+  .styled-select select:-moz-focusring {
+    color: transparent;
+    text-shadow: 0 0 0 #000;
+  }
+  .styled-select select::-ms-expand {
+    display: none;
+  }
+  .styled-select i {
+    position: absolute;
+    right: 10px;
+    top: 12px;
+    pointer-events: none;
+  }
 }
 
-#dashboard-upper-content{
-  .editor-wrapper {
-    padding-bottom: 70px;
+// 940px grid without margin
+// -------------------------
+@gridColumnWidthNoMargin:         60px;
```

Comment:
i had to create my own grid as bootstrap 2 grids have 20px margin but i need full-sized borders on the left size - but still want to take adavantage of `span5` and `span7`